src/nyc_trees/apps/home/urls.py

```diff
@@ -17,7 +17,9 @@
     url(r'^jobs/(?P<job_id>\d+)/$', r.retrieve_job_status,
         name='retrieve_job_status'),
 
-    url(r'^training/$', r.training_list_page),
+    url(r'^training/$', r.training_list_page, name="training_list_page"),
+
+    url(r'^training/groups_to_follow/$', r.groups_to_follow),
```

Comment:
This is not a flatpages thing, we're hijacking certain `training/` urls to make custom, regular django pages and exposing the rest as flat pages.

IIRC, the django convention is to require all urls to end in a trailing slash, and use the `APPEND_SLASH` middleware to force requests to have one. You verified that a `404` happened?

https://docs.djangoproject.com/en/1.7/ref/settings/#append-slash